hazardous/data/tests/test_seer.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+import pandas as pd
+from pandas.testing import assert_frame_equal
+
+from hazardous.data._seer import load_seer
+
+WD_PATH = Path("hazardous/data/tests").resolve()
+
+
+def test_load_seer():
+    input_path = WD_PATH / "seer_sample.txt"
+    expected_output_path = WD_PATH / "expected_preprocessed_seer_sample.csv"
+
+    X, y = load_seer(input_path)
+    assert X.shape == (3, 28)
+
+    expected_y = pd.DataFrame(
+        dict(
+            event=[1.0, 0.0, 1.0],
+            duration=[81, 7, 28],
+        )
+    )
+    assert_frame_equal(y, expected_y)
+    expected_X = pd.read_csv(expected_output_path)
+    valid_columns = X.columns[~X.columns.str.contains("xxx")]
+    assert len(valid_columns) == 25
+    assert_frame_equal(X[valid_columns], expected_X[valid_columns])
```

Comment:
Please test that `NUMERIC_COLUMN_NAMES` have all numerical dtypes and `CATEGORICAL_COLUMN_NAMES` have categorical dtypes.

Also we could add another test that would use the raw Seer text file and check thinkgs like:

- the number of rows;
- the cardinality of categorical columns and in particular checking that the number of unique values for categorical columns is below some threshold (e.g. 10% of the total number of rows);
- the value counts for each categorical columns;
- the numerical dtypes infers by pandas;
- the mean, median, 25%, 75% percentiles, min and max values of numerical columns;
- that columns that are neither numerical nor categorical have a object dtype (or arrow string in the future but let's not introduce a dependency on pyarrow for now to keep jupyterlite compat in hazardous);
- the number of missing values per columns.

This would help detect that possible behavioral changes in future versions of pandas do not go unnoticed.